docs/_content/standaard/verzoeken/index.md

```diff
@@ -0,0 +1,69 @@
+* Zuiverheid: er ontstaat een duidelijk onderscheid tussen 'klantorders' en interne 'productie-/pickorders'.
+* Eenvoud: het onderscheid tussen hoofd- en deelzaken kan op termijn vervallen, en wordt vervangen door onderscheid tussen verzoeken en zaken. Uitzoeken hoe het zit met regiezaken.
+* Verantwoording: de informatie die bij de intake aanwezig was, is de informatie die hoort bij het verzoek. Daarmee is meteen duidelijk informatie die aan een zaak (of een andere behandelwijze) is gekoppeld, na de initiële aanvraag is toegevoegd.
+
+# Intake
+Op een verzoek wordt een intake gedaan. Daarbij wordt het verzoek niet inhoudelijk beoordeeld. Wel wordt gekeken of de aangevraagde producten geleverd kunnen worden, en of alle daarvoor benodigde gegevens zijn ingevuld. Na de intake is het verzoek 'in behandeling genomen' danwel 'buiten behandeling gesteld'. De status van een verzoek kan in de PIP getoond worden, maar let op: dat dit verwarrend kan zijn als ook statussen van zaken getoond worden. Het is aan consumers om op te lossen.
+
+De initieel mogelijke statussen reflecteren de status van de intake: nieuw, ingetrokken, in intake/behandeling, in behandeling genomen, afgewezen. Nadat een verzoek in behandeling is genomen (of afgewezen) kan het omwille van de verantwoording niet meer worden gewijzigd.
+
+De intake van een verzoek kan zowel automatisch als handmatig plaatsvinden. De API moet beide ondersteunen. De consumers moeten zelf bepalen hoe de status van verzoek en zaak wordt gebruikt/gepresenteerd.
+
+Nadat een verzoek is ingediend kan het niet meer worden gewijzigd. Wel kan de status worden gewijzigd n.a.v. de intake en kunnen er relaties worden gelegd met andere verzoeken, zaken, informatieobjecten, contacten, etc.
+
+# Relatie met zaken
+Een verzoek kan resulteren in één of meerdere zaken of andere activiteiten (d.w.z. niet-zaakgericht), danwel direct worden afgehandeld, bijvoorbeeld door levering van een informatiebrochure na aanvraag daarvan. Bovendien kan een verzoek worden gekoppeld aan een reeds lopende zaak. Denk bijvoorbeeld aan een Melding Openbare Ruimte die reeds eerder door een andere klant is gemeld, en waarvoor reeds een zaak loopt. Denk hierbij ook aan een verzoek tot intrekken van een eerder verzoek (dat, indien naar aanleiding van dat eerdere verzoek reeds een zaak was gestart, als tweede (of derde, vierde enz.) verzoek aan die zaak moet worden gekoppeld)
```

Comment:
> Tegelijkertijd kunnen meerdere verzoeken worden afgehandeld in één zaak.

> Bovendien kan een verzoek worden gekoppeld aan een reeds lopende zaak. Denk bijvoorbeeld aan een Melding Openbare Ruimte die reeds eerder door een andere klant is gemeld, en waarvoor reeds een zaak loopt.

Hoe ziet dit er in de praktijk uit? Een melding is immers een verzoek en leidt direct tot een "verzoeknummer". Vervalt dan later het tweede verzoeknummer als het tweede verzoek aan dezelfde zaak wordt gekoppeld.